mondey_backend/src/mondey_backend/routers/scores.py

```diff
@@ -0,0 +1,242 @@
+from __future__ import annotations
+
+from datetime import datetime
+from datetime import timedelta
+from enum import Enum
+from typing import cast
+
+import numpy as np
+from sqlmodel import select
+
+from ..dependencies import SessionDep
+from ..models.children import Child
+from ..models.milestones import MilestoneAgeScore
+from ..models.milestones import MilestoneAgeScoreCollection
+from ..models.milestones import MilestoneAnswerSession
+from ..models.milestones import MilestoneGroupAgeScore
+from ..models.milestones import MilestoneGroupAgeScoreCollection
+from .statistics import calculate_milestone_statistics_by_age
+from .statistics import calculate_milestonegroup_statistics_by_age
+from .utils import get_child_age_in_months
+
+
+class TrafficLight(Enum):
+    """
+    Enum for the trafficlight feedback.
+    Includes -1 for red, 0 for yellow, and 1 for green.
+    Invalid is -2 and is included for edge cases.
+    """
+
+    invalid = -3
+    red = -2
+    yellowWithCaveat = -1
+    yellow = 0
+    greenWithCaveat = 1
+    green = 2
+
+
+def compute_feedback_simple(
```

Comment:
I've done it this way because we have to convert the feedback into numbers at some point (unless I miss something) to have the frontend work with them, and I figured that it would be the most contained way to do this here. The enum serves essentially for naming values. Otherwise we need to convert  it in the endpoints at the latest, which is a bit messy.

Reply:
I think that you can use the enum throughout, and the frontend response will automatically output the underlying int values in the emitted json (I haven't tried to confirm this though)